Cartfile

```diff
@@ -13,3 +13,4 @@ github "st3fan/SWXMLHash"           "xcode-7.0"        # So we get Swift 2.0 fix
 github "DaveWoodCom/XCGLogger"      "swift_2.0"        # Previously: swift_1.2
 github "sleroux/google-breakpad-ios" "master"
 github "adjust/ios_sdk"             "v4.3.0"
+github "AgileBits/onepassword-extension" "add-framework-support"
```

Comment:
Can you pin this to their 1.6.4 tag? Which is the most recent version in the add-framework-support branch?